recipes/binutils/all/test_v1_package/conanfile.py

```diff
@@ -1,4 +1,5 @@
 from conans import ConanFile, tools
+from conan.tools.layout import basic_layout
```

Comment:
Please, revert it, we don't use layout for test_v1_package, it should test the legacy of Conan 1.x, not new features.